tests/test_frozenset.py

```diff
@@ -0,0 +1,8 @@
+import pandas as pd
+
+s = pd.Series(["A", "B", "AB"])
+
+s.apply(tuple)
+s.apply(list)
+s.apply(set)
+s.apply(frozenset)
```

Comment:
This should probably be a function in test_series.py

Reply:
Removed the extra test file and put it as a function in test_series.py!

Reply:
I know you are not changing the output, but we try to use `check` and `assert_type` in all new tests
`check(assert_type(s.apply(frozenset), "pd.Series[dtype]" , pd.Series)`

Reply:
Just fixed the tests.